### DISCUSSION TITLE
啊这，怎么成乐子了

### DISCUSSION THREAD

**Discussion by Arondight** [Arondight/Adachi-BOT#629] [General]
几天没上论坛，狐狸的外号成乐子了……~果盘我还能理解~

不知道写代码外还有啥好营生，不太想写了

**Comment by mark9804** [comment]
![image](https://user-images.githubusercontent.com/9006264/151997732-fcaf008c-f5b1-4150-b756-d03d80cbdb6c.png)

**Comment by Arondight** [comment]
![FB9FC723-0AFE-4BE2-A60C-A3330216D84F](https://user-images.githubusercontent.com/13673702/152391696-2de19c85-309f-425c-a68d-8d88782f1771.jpeg)

**Reply by Arondight** [reply]
我脑测了一下，千岩讨龙乐子，似乎还行
